timemachine/md/minimizer.py

```diff
@@ -235,23 +231,18 @@ def equilibrate_host(
     hgt = topology.HostGuestTopology(host_bps, top)
 
     # setup the parameter handlers for the ligand
-    tuples: List[Tuple[Any, List[Any]]] = [
-        (hgt.parameterize_harmonic_bond, [ff.hb_handle]),
-        (hgt.parameterize_harmonic_angle, [ff.ha_handle]),
-        (hgt.parameterize_periodic_torsion, [ff.pt_handle, ff.it_handle]),
-        (hgt.parameterize_nonbonded, [ff.q_handle, ff.lj_handle]),
+    ff_params = ff.get_params()
+    hb_params, hb_potential = hgt.parameterize_harmonic_bond(ff_params.hb_params)
+
+    params_potential_pairs = [
+        (hb_params, hb_potential),
+        hgt.parameterize_harmonic_angle(ff_params.ha_params),
```

Comment:
Curiosity: is the different syntax for harmonic bonds in this list significant?

(Is this mostly to document the tuple contents of the subsequent ones?)

Reply:
This is just to have a named reference to `hb_potential`, since it's used again a few lines below to get the bond list.